src/SonarQube.Client/Models/Properties.cs

```diff
@@ -18,21 +18,29 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-using System.Diagnostics;
 using Newtonsoft.Json;
 
-namespace SonarLint.VisualStudio.Integration.Service
+namespace SonarQube.Client.Models
 {
-    [DebuggerDisplay("Key = {key}, Value = {value}")]
-    internal class ServerProperty
+    public class SonarQubeProperty
     {
-        #region Known properties
+        public string Key { get; }
+        public string Value { get; }
 
-        public const string TestProjectRegexKey = "sonar.cs.msbuild.testProjectPattern";
-        public const string TestProjectRegexDefaultValue = @"[^\\]*test[^\\]*$";
+        public SonarQubeProperty(string key, string value)
+        {
+            Key = key;
+            Value = value;
+        }
 
-        #endregion
+        public static SonarQubeProperty FromDto(PropertyDTO dto)
+        {
+            return new SonarQubeProperty(dto.Key, dto.Value);
+        }
+    }
 
+    public class PropertyDTO
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename class 'PropertyDTO' to match camel case naming rules, consider using 'PropertyDto'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS101)